Task 3/Task 3.1/Text Analysis/Classes/Data.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace Text_Analysis.Classes
+{
+    public class Data
+    {
+        public Data(int quantity)
+        {
+            QuantityWordsData = new Dictionary<string, int> ();
+            PercentWordsData = new Dictionary<string, double>();
+            QuantityOfWords = CheckQuantity(quantity);
+        }
+
+        public Dictionary<string, int> QuantityWordsData { get; private set; }
+
+        public Dictionary<string, double> PercentWordsData { get; private set; }
```

Comment:
You could save a stats class object as value in a single dictionary

Reply:
`Dictionary<string, Stats>`

Reply:
Haven't seen your comment while did this))